README.md

```diff
@@ -34,7 +34,7 @@ Also known as [盆栽(bonsai)](https://en.wikipedia.org/wiki/Bonsai) 🌳
    ```
 1. Make shells installed by nix into a login shell
    ```bash
-   nix run 'github:kachick/dotfiles#sudo_uinit' -- --dry_run=false
+   sudo "$(which nix)" run 'github:kachick/dotfiles#uinit' -- --user="$(whoami)" --dry_run=false
```

Comment:
This ways removes workaround in unix, but darwin 13 and 14 still made related warning... 🤔 

https://github.com/kachick/dotfiles/actions/runs/8167128551/job/22327025439?pr=434
https://github.com/kachick/dotfiles/actions/runs/8167128551/job/22327025567?pr=434

```
warning: $HOME ('/Users/runner') is not owned by you, falling back to the one defined in the 'passwd' file ('/var/root')
```

https://github.com/NixOS/nix/issues/6834 👀

Reply:
✅ https://github.com/kachick/dotfiles/pull/434/commits/3b50412a9c4a0ac15c08cfc59ba424ca933fa4e4